src/subcommand.go

```diff
@@ -71,4 +98,22 @@ func CommandReadRun() {
 
 func loadIndex(indexDir string) {
 	fmt.Println("loadIndex in commands.go is unimplemented")
-}
\ No newline at end of file
+}
+
+func parseDataType(str string) (pb.DataType, error) {
```

Comment:
for this and the parseSourceType function I think we can use https://pkg.go.dev/google.golang.org/protobuf/internal/testprotos/enums#Enum.Descriptor or Descriptor() from https://github.com/accretional/semantifly/blob/main/src/proto/accretional.com/semantifly/proto/index.pb.go which should allow us to eliminate this boilerplate string->enum conversion code

Reply:
Changed it to use the enum values directly